integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/predicate/MatchSearchPredicateIT.java

```diff
@@ -69,6 +69,35 @@ public void setup() {
 		initData();
 	}
 
+	@Test
+	public void match_id() {
```

Comment:
We tried to create one test class per type of predicate: `MatchSearchPredicateIT`, `RangeSearchPredicateIT`, etc.

Could you please move this to a different test class?

The class can be quite simple, much simpler than this one, because you don't really have to test multiple types of fields. Maybe you could take inspiration from what @fax4ever did here: https://github.com/hibernate/hibernate-search/commit/d0d1c3e1c8e94704451e976f95553cb5071bf694